server/channels/app/properties/property_field.go

```diff
@@ -21,6 +21,10 @@ func (ps *PropertyService) GetPropertyFields(groupID string, ids []string) ([]*m
 	return ps.fieldStore.GetMany(groupID, ids)
 }
 
+func (ps *PropertyService) GetPropertyFieldByName(groupID, targetID, name string) (*model.PropertyField, error) {
```

Comment:
What is the `targetID` here? From my local setup, it seems like empty for my Property Fields.

Reply:
And that's correct. Target&TargetID are intended to point to an entity in case the feature has entity-wide properties (imagine property fields belonging to playbooks, each would point to its playbook using those fields).

For the case of CPA, we currently only have system-wide properties, so no target needed.

Reply:
Oh gotcha, thanks!